parlai/crowdsourcing/tasks/qa_data_collection/run.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import os
+import sys
+from mephisto.operations.operator import Operator
+from mephisto.tools.scripts import load_db_and_process_config
+from mephisto.abstractions.blueprints.parlai_chat.parlai_chat_blueprint import (
+    BLUEPRINT_TYPE,
+    SharedParlAITaskState,
+)
+
+import hydra
+from omegaconf import DictConfig, MISSING
+from dataclasses import dataclass, field
+from typing import List, Any
+
+from parlai.tasks.squad.agents import DefaultTeacher
+
+# This needs to be properly streamlined
+class SquadDataLoader(DefaultTeacher):
+
+    def __init__(self, opt):
+        super().__init__(opt)
+
+    def act(self):
+        data = super().act()
+        # SQuAD returns passage and question both, only passage required for task
+        data.force_set('text', '\n'.join(data['text'].split('\n')[:-1]))
+        return data
+
+TASK_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+
+defaults = [
+    {"mephisto/blueprint": BLUEPRINT_TYPE},
+    {"mephisto/architect": "local"},
+    {"mephisto/provider": "mock"},
+    "conf/base",
+    {"conf": "example"},
+    {"conf": "dataloader_squad"}
+]
+
+from mephisto.operations.hydra_config import RunScriptConfig, register_script_config
+
+@dataclass
+class DataLoaderConfig:
```

Comment:
See notes above for how to simplify this class - I think only 2 params here are truly necessary, `datatype` and `task`